azerothcore/mod-bg-item-reward
# Added Eye of the Storm BG, Added Tie status for the rest of BGs.

You are most welcome. :=D

## Changes Proposed:
- Added EOTS (Eye of the Storm) BG (Battleground).
- Added Tie status to all other BGs including EOTS.
 
## Issues Addressed:
- Missing Eye of the Storm Battleground Item Reward.
- Missing Tie Reward for the rest of Battlegrounds, including Eye of the Storm.

## SOURCE:
- https://www.wowhead.com/items/currency

## Tests Performed:
- The AzerothCore source compiles without errors with this and other modules in the modules folder.
- I have not tested in-game, someone else probably should.
- Using Windows 11 Pro with an 8th generation Core i5, an ASUS ROG Strix GTX 1070 Ti, and a 1TB Samsung 970 EVO NVME SSD.

## How to Test the Changes:
1. Play a Battleground and Win, then check currency.
2. Play a Battleground and Lose, then check currency.
3. Play a Battleground and Tie, then check currency.
